lib/scenic_font_press_start_2p.ex

```diff
@@ -1,20 +1,22 @@
 defmodule ScenicFontPressStart2p do
 
+  {:ok, hash} =
+    Application.app_dir(:scenic_font_press_start_2p, ["priv", "PressStart2P.ttf"])
+    |> Scenic.Cache.Hash.file(:sha)
+
+  @hash hash
```

Comment:
```
@hash Scenic.Cache.Hash.file!(path(), :sha)
```

Reply:
You cant call a function thats being compiled when defining a module attribute.

Reply:
darn. Well, then how about using the bang version of `file!` to avoid the `{:ok, hash}` match.